Formula/jupyterlab.rb

```diff
@@ -335,18 +272,31 @@ class Jupyterlab < Formula
   end
 
   def install
-    venv = virtualenv_create(libexec, Formula["python@3.9"].opt_bin/"python3")
+    xy = "3.10"
+    ENV.prepend_create_path "PYTHONPATH", Formula["ipython"].opt_libexec/"vendor/lib/python#{xy}/site-packages"
+    ENV.prepend_create_path "PYTHONPATH", Formula["ipython"].opt_libexec/"lib/python#{xy}/site-packages"
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python#{xy}/site-packages"
+    venv = virtualenv_create(libexec, Formula["python@#{xy}"].opt_bin/"python3")
     ENV["JUPYTER_PATH"] = etc/"jupyter"
 
-    # gather packages to link based on options
-    linked = %w[jupyter-core jupyter-client nbformat ipykernel jupyter-console nbconvert notebook]
+    # install other resources
+    linked = %w[jupyter-core jupyter-client nbformat jupyter-console nbconvert notebook]
     dependencies = resources.map(&:name).to_set - linked
     dependencies -= ["appnope"] if OS.linux?
     dependencies.each do |r|
       venv.pip_install resource(r)
     end
+
+    # install and link JupyterLab
     venv.pip_install_and_link linked
     venv.pip_install_and_link buildpath
+    bin.glob("*") do |file|
+      file.unlink
+      file.write_env_script libexec/"bin"/file.basename, {
+        JUPYTER_PATH: ENV["JUPYTER_PATH"],
+        PYTHONPATH:   ENV["PYTHONPATH"] + "${PYTHONPATH:+:}$PYTHONPATH",
```

Comment:
```suggestion
        PYTHONPATH:   ENV["PYTHONPATH"] + "${PYTHONPATH:+:}${PYTHONPATH}",
```
We might want to add `HOMEBREW_PREFIX/site_packages` here too so that you can use Homebrew-installed Python modules in Jupyter. Though `virtualenv_create` might already do that.